lms/templates/courseware/progress_graph.js

```diff
@@ -128,13 +202,65 @@ $(function () {
     markings.push({yaxis: {from: ascending_grades[i], to: ascending_grades[i+1]}, color: colors[(i-1) % colors.length]});
 
   var options = {
-    series: {stack: true,
-              lines: {show: false, steps: false },
-              bars: {show: true, barWidth: 0.8, align: 'center', lineWidth: 0, fill: .8 },},
-    xaxis: {tickLength: 0, min: 0.0, max: ${tickIndex - sectionSpacer}, ticks: ticks, labelAngle: 90},
-    yaxis: {ticks: grade_cutoff_ticks, min: 0.0, max: 1.0, labelWidth: 100},
-    grid: { hoverable: true, clickable: true, borderWidth: 1, markings: markings },
-    legend: {show: false},
+    series: {
+        stack: true,
+        lines: {
+            show: false,
+            steps: false
+        },
+        bars: {
+            show: true,
+            barWidth: 0.8,
+            align: 'center',
+            lineWidth: 0,
+            fill: .8
+        }
+    },
+    xaxis: {
+        tickLength: 0,
+        min: 0.0,
+        max: ${tickIndex - sectionSpacer},
+        ticks: function() {
+            for (var i = 0; i < ticks.length; i++) {
+                var tickLabel = edx.HtmlUtils.joinHtml(
+                    // The very last tick will be for the total, and it usually is composed of a number of different
+                    // grading types. To help clarify, do NOT make the label ("Total") aria-hidden in that case.
+                    edx.HtmlUtils.HTML(i < ticks.length - 1 ? '<span aria-hidden="true">' : '<span>'),
+                    ticks[i][1],
+                    edx.HtmlUtils.HTML('</span>')
+                );
+                var elementTooltips = yAxisTooltips[ticks[i][0]];
+                if (elementTooltips) {
+                    for (var tooltipIndex = 0; tooltipIndex < elementTooltips.length; tooltipIndex++) {
+                        tickLabel = edx.HtmlUtils.joinHtml(
+                            tickLabel,
+                            edx.HtmlUtils.HTML('<span class="sr">'),
+                            elementTooltips[tooltipIndex],
+                            edx.HtmlUtils.HTML('</span>')
```

Comment:
@cahrens can we append a ```<br>``` element here? Windows screen readers read multiple spans on one big run on sentence, so the Total grade region gets really hard to understand.